### DISCUSSION TITLE
Change default project?

### DISCUSSION THREAD

**Discussion by jrcasey** [marcusolsson/obsidian-projects#761] [Help] [answered]
When I open projects, one of my projects (let's call it "Project X") appears. That's fine. But when I switch to a different project ("Project Y") and pull up a note, when I return to the projects view (for example using the back button) it reverts to Project X. This is not the end of the world but it's surprisingly annoying when I'm rapid-fire editing notes in Project Y. Is there any way to change this behavior? Thanks and great job with the plugin!

**Comment by haShinui** [comment, marked as answer]
So its an easy fix, just click the free dots at the top right of projects view, then edit project and check or uncheck "set as default".

**Reply by jrcasey** [reply]
Thank you! 🙏
